main.cpp

```diff
@@ -1010,13 +730,371 @@ Part 1e - Step 19: Request a review
+    //     1) Generate lift
+    void generateLift();
+    //     2) Provide stability
+    void stabilize();
+    //     3) Adjust to various conditions and scenarios
+    void adjustToTurbulance();
+}; 
+
+struct Engines 
+{
+//     5 properties:
+//         1) Number of engines (int)
+    int numEngines = 2;
+//         2) Power output (float)
+    float powerOutput = 30.f;
+//         3) Fuel efficiency (float)
+    float fuelEfficiency = 10.f;
+//         4) Emissions (float)
+    float emissions = 20.f;
+//         5) Size (int)
+    int size = 20;
+
+    struct Turbines
+    {
+        int lowPressureLenth = 84; 
+        int highPressureWidth = 93;
+        std::string brand = "GE";
+        std::string model = "CF3942";
+        std::string material ="Titanium";
+
+        void thrust(std::string whichTurbine);
+        void efficiency(std::string targetSpeed);
+        void noiseLevel(std::string whichTurbine);
+    };
 
 
+//     3 things it can do:
+//         1) Provide thrust
+    void engageThrust(Turbines turbines);
+//         2) Generate elecricity
+    void increasePower(Turbines turbines, int amountOfIncrease);
+//         3) Control speed and altitude
+    void controlSpeedAndAltitude(Turbines turbines, int amountOfChange);
+
+    Turbines engaged;
+};
+
+struct CargoSpace
+{
+    // 5 properties:
+    //     1) Volume (float)
+    float volume = 64.f;
+    //     2) Weight (int)
+    int weight = 1300;
+    //     3) Temperature (int)
+    int temperature = 75;
+    //     4) Capacity (int)
+    int capacity = 400;
+    //     5) Loading time (float)
+    float loadingTime = 45;
+    // 3 things it can do:
+    //     1) Transport
+    void transportCargo();
+    //     2) Store
+    void storeCargo();
+    //     3) Secure
+    void secureCargo();
+};
+
+struct Tail
+{
+    // 5 properties:
+    //     1) Tail length (int)
+    int length = 55;
+    //     2) Tail width (int)
+    int width = 85;
+    //     3) Elevator angle (float)
+    float elevatorAngle = 45.f;
+    //     4) Stabilizer angle (float)
+    float stabilizerAngle = 45.f;
+    //     5) Yaw control width (int)
+    int yawControlWidth = 30;
+    // 3 things it can do:
+    //     1) Create stability
+    void stabilize();
+    //     2) Create lift
+    void createLift();
+    //     3) Control side to side movement
+    void controlSideToSide();
+};
+
+struct Fuselage
+{
+    // 5 properties:
+    //     1) Length of fuselage (int)
+    int length = 30;
+    //     2) Width of fuselage (int)
+    int width = 15;
+    //     3) Height of fuselage (float)
+    float height = 19.f;
+    //     4) Capacity of fuselage (int)
+    int capacity = 59;
+    //     5) Drag from fuselage (float)
+    float drag = 32.2f;
+    // 3 things it can do:
+    //     1) Enclose passenger cabin and cargo
+    void encloseCabin();
+    //     2) Support wings and tail
+    void supportWingsAndTail();
+    //     3) Maintain internal pressure
+    void pressureControlCheck();
```

Comment:
I'm marking this because:
1. your plain-english description is not an action, but is an ability or describes a state OR
2. the plain-english description is fine but the function name doesn't match OR
3. The way you wrote the name doesn't conform to the course coding standard:
    use `camelCase` for variable and function names.
    Type Names begin with a Capitalized first letter. All other symbols begin with a lower-case first letter.
- [ ] revise